jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/EditUserProfileDto.java

```diff
@@ -248,6 +245,21 @@ public UserProfileDto getUserProfileDto() {
     }
 
     /**
+     * Set user profile version
+     * @param version - profile version
+     */
+    public void setUserProfileVersion(Long version) {
```

Comment:
I don't think so. This is a common code style. Look, every getter and setter has its own Javadoc.